product-service/src/functions/catalogBatchProcess/handler.ts

```diff
@@ -0,0 +1,39 @@
+import 'source-map-support/register';
+
+import { SNS } from 'aws-sdk';
+import { SQSEvent } from 'aws-lambda';
+import { create } from '@libs/db';
+import { middyfy } from '@libs/lambda';
+import { formatJSONResponse } from '@libs/apiGateway';
+
+export const catalogBatchProcess = async (event: SQSEvent) => {
+  console.info(event);
+
+  const sns = new SNS({ region: 'eu-west-1' });
+
+  try {
+    for (let record of event.Records) {
+      const product = JSON.parse(record.body);
+      await create(product);
+      await sns
+        .publish({
+          Subject: `${product.title} has been created.`,
+          Message: JSON.stringify(product),
+          TopicArn: process.env.SNS_ARN,
+          MessageAttributes: {
+            price: {
+              DataType: 'Number',
+              StringValue: `${Number(product.price)}`,
```

Comment:
Why are you converting to a number and then to a string?

Reply:
Good point. I don't know. :) But I know that it works in such way. I will check if I can replace it by `StringValue: product.price` since it is already a string